tools/testdata-scripts/src/bin/make-testdata.rs

```diff
@@ -33,44 +18,48 @@ fn main() {
         .init()
         .unwrap();
 
+    let out_dir = Path::new(concat!(
+        core::env!("CARGO_MANIFEST_DIR"),
+        "/../../provider/testdata/data/"
+    ));
```

Comment:
Thought: It's messy to have paths go outside of the root directory of a crate. Is there a way you can avoid that? Before we could use a path relative to `icu_testdata` but dropping that dependency was a nice improvement since it means we can run make-testdata without databake buidling.

Reply:
ok, for a no-publish crate.